Metalama.Testing.AspectTesting/TestResult.cs

```diff
@@ -411,7 +463,7 @@ public IReadOnlyList<SyntaxTree> GetTestOutputsWithDiagnostics()
                     CSharpSyntaxTree.Create(
                         consolidatedCompilationUnit,
                         path: Path.GetFileName(
-                            outputSyntaxTree.InputPath
+                            outputSyntaxTree.InputPath ?? outputSyntaxTree.OutputDocument?.FilePath
```

Comment:
Same reuse of ` outputSyntaxTree.InputPath ?? outputSyntaxTree.OutputDocument?.FilePath`